<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Techno-311" failures="1" tests="1" name="AidGetIndividualId" time="1.124" errors="0" timestamp="16 Nov 2022 08:49:57 GMT">
  <testcase classname="io.mosip.testscripts.SimplePost" name="" time="1.118">
    <failure type="io.mosip.admin.fw.util.AdminTestException" message="Failed at output validation">
      <![CDATA[io.mosip.admin.fw.util.AdminTestException: Failed at output validation
	at io.mosip.testscripts.SimplePost.test(SimplePost.java:134)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:164)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:111)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!--  -->
</testsuite> <!-- AidGetIndividualId -->
